### PR TITLE
Fixing #530 - Adding support for SILKY_EXPLAIN_FLAGS

### PR DESCRIPTION
**Pull request by baruchoxman**
Adding support for additional flags that can be passed as part of the "analyze" query, for more explicit analysis.

### CODE REVIEW

File: silk/sql.py [hunk 1]
```diff
@@ -34,10 +34,11 @@ def _explain_query(connection, q, params):
             # Work around some DB engines not supporting analyze option
             try:
                 prefix = connection.ops.explain_query_prefix(
-                    analyze=True
+                    analyze=True, **(SilkyConfig().SILKY_EXPLAIN_FLAGS or {})
                 )
             except ValueError as error:
-                if str(error) == "Unknown options: analyze":
+                error_str = str(error)
+                if error_str.startswith("Unknown options:") and "analyze" in error_str:
```
Comment (liorzmetis) [inline silk/sql.py]: When error occurred it would be better to show which option was a problem and append the error to the logger.

**Review by liorzmetis — CHANGES_REQUESTED** [review]
looks good

File: silk/sql.py (hunk 1 above)
Comment (baruchoxman) [reply silk/sql.py]: Thanks, fixed

**Review by liorzmetis — APPROVED** [review]

**Review by Archmonger — APPROVED** [review]

**Comment by albertyw** [comment]
Merged.  Thanks for the PR.

**Pull request merged**